orm_lib/inc/drogon/orm/Exception.h

```diff
@@ -209,6 +209,18 @@ class InternalError : public DrogonDbException, public std::logic_error
     DROGON_EXPORT explicit InternalError(const std::string &);
 };
 
+/// Timeout error in when executing the SQL statement.
+class TimeoutError : public DrogonDbException, public std::logic_error
+{
+    virtual const std::exception &base() const noexcept override
+    {
+        return *this;
+    }
+
+  public:
+    DROGON_EXPORT explicit TimeoutError(const std::string &);
```

Comment:
Maybe move this up into class deceleration? In case we need to add new methods to the class.

Reply:
The other exceptions are all the same.

Reply:
I see, Let's change this if we need to add methods to other exceptions in the future